runtime/doc/vim_diff.txt

```diff
@@ -74,7 +74,10 @@ the differences.
 
 
 Default Mappings:					*default-mappings*
-  nnoremap Y y$
+
+- nnoremap Y y$
+- <C-L> clears search highlights |:nohlsearch| and updates diffs
+  |:diffupdate|
```

Comment:
nit: i think we should not have the bullet points, they end up being noisy in this help doc because of the sheer amount of items. Also for listing mappings they are distracting because mappings are a precise series of chars.

I think we should just list the literal mappings here. Explain the semantics at `:help CTRL-L`